src/pages/Home.tsx

```diff
@@ -47,28 +47,34 @@ const watchSectionData: Array<any> = [
     title: "Evolution of human species",
   },
 ];
-
+const subjectIcons = {
```

Comment:
@sagarkoshti1990 are the subjects hard-coded?
we might have subjects other than the above-mentioned subjects